app/views/events/edit.html.erb

```diff
@@ -3,4 +3,4 @@
 <%= render 'form', event: @event %>
 
 <%= link_to 'Show', @event %> |
```

Comment:
Remember about translations. Please check also what this auto correction bot says about your code.